packages/web-components/src/components/ic-data-table/ic-data-table.stories.mdx

```diff
@@ -5,1974 +5,661 @@ import {
+  DATA,
+  Basic,
+  Embedded,
+  Dense,
+  Spacious,
+  Scrollable,
+  RowHeaders,
+  Sort,
+  SortOptions,
+  Pagination,
+  ColumnOverrides,
+  RowOverrides,
+  CellOverrides,
+  LinksHTMLElements,
+} from "./story-data";
 import readme from "./readme.md";
 
-<Meta title="Web Components/Data table" component="ic-unstable-data-table" />
-
-<Description markdown={readme} />
-
-### Basic table
-
-<Canvas>
-  <Story name="Basic table">
-    {html` <script>
-        dataTable = document.querySelector("ic-unstable-data-table");
-        dataTable.columns = [
-          {
-            key: "firstName",
-            title: "First name",
-            dataType: "string",
-          },
-          {
-            key: "lastName",
-            title: "Last name",
-            dataType: "string",
-          },
-          {
-            key: "age",
-            title: "Age",
-            dataType: "number",
-          },
-          {
-            key: "jobTitle",
-            title: "Job title",
-            dataType: "string",
-          },
-          {
-            key: "address",
-            title: "Address",
-            dataType: "address",
-          },
-        ];
-        dataTable.data = [
-          {
-            firstName: "Joe",
-            lastName: "Bloggs",
-            age: 30,
-            jobTitle: "Developer",
-            address: "1 Main Street, Town, County, Postcode",
-          },
-          {
-            firstName: "Sarah",
-            lastName: "Smith",
-            age: 28,
-            jobTitle: "Analyst",
-            address: "2 Main Street, Town, Country, Postcode",
-          },
-          {
-            firstName: "Mark",
-            lastName: "Owens",
-            age: 45,
-            jobTitle: "Team Lead",
-            address: "12 Key Street, Town, Country, Postcode",
-          },
-          {
-            firstName: "Naomi",
-            lastName: "Thomas",
-            age: 32,
-            jobTitle: "Developer",
-            address: "8 Side Street, Town, Country, Postcode",
-          },
-          {
-            firstName: "Luke",
-            lastName: "Ashford",
-            age: 18,
-            jobTitle: "Junior Developer",
-            address: "5 New Street, Town, Country, Postcode",
-          },
-        ];
-      </script>
-      <ic-unstable-data-table caption="basic table"></ic-unstable-data-table>`}
-  </Story>
+<Meta
+  title="Web Components/Data Table"
+  component="ic-unstable-data-table"
+  parameters={{
+    componentAPI: {
+      data: readme,
+    },
+  }}
+/>
+
+# [Canary]: Data Table
+
+Use the data table component to display data in a tabular form.
+
+There are two required props for data table:
+
+- Caption: `string`
+- Columns: `IcUnstableDataTableColumnObject[]`
+
+Click the 'Component API' tab to view all the available props, events and slots for Data Tables.
+
+To import the data table web component into your application, import `@ukic/canary-web-components` into your application.
```

Comment:
The previous comment about wording in the react storybook would apply here too

Reply:
I've changed it to 'To use the data table component.... The term `web component` has been removed from both making it more generic.